rules/duo_rules/duo_user_anomalous_push.py

```diff
@@ -0,0 +1,22 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return event.get("reason") == "anomalous_push" and event.get("result") == "denied"
+
+
+def title(event):
+    # pylint: disable=line-too-long
```

Comment:
No need to disable the line too long. `make fmt` will format the title as needed

Reply:
I couldn't get fmt to reduce the line, but I did break it up so it's short enough now